python/instance_test.py

```diff
@@ -145,9 +149,11 @@ def test_serialize(self):
             ],
         )
```

Comment:
I prefer making copying the instances vs. a setUp method; it's easier to read the test by itself and verify that it's correct.

https://testing.googleblog.com/2019/12/testing-on-toilet-tests-too-dry-make.html